docs/events/event_actions.md

```diff
@@ -209,3 +209,43 @@ updated default_value from the state.
 Without the `key` set, the slider would always display the original
 `settled_value` after a page reload, instead of its current value.
 ```
+
+## Temporal Events
+
+_Added in v0.5.0_
```

Comment:
it was actually added in [v0.6.6](https://github.com/reflex-dev/reflex/releases/tag/v0.6.6)

Next time check the reflex release notes on https://github.com/reflex-dev/reflex/releases/ when determining what version a feature was added in